pkgs/development/libraries/libfprint/default.nix

```diff
@@ -44,17 +33,15 @@ stdenv.mkDerivation rec {
     gtk-doc
     docbook_xsl
     docbook_xml_dtd_43
+    gobject-introspection
   ];
 
   buildInputs = [
-    libusb1 # drop in 2.0 for gusb
+    gusb
     pixman
     glib
     nss
-    gtk3
-  ]
-  ++ stdenv.lib.optional thinkpad openssl
-  ;
+  ];
 
   mesonFlags = [
     "-Dudev_rules_dir=${placeholder "out"}/lib/udev/rules.d"
```

Comment:
WARNING: Unknown options: "x11-examples"

Reply:
substituteStream(): WARNING: pattern '/bin/echo' doesn't match anything in file 'libfprint/meson.build'